packages/devtools_app/lib/src/vm_service_wrapper.dart

```diff
@@ -221,6 +221,44 @@ class VmServiceWrapper implements VmService {
     }
   }
 
+  //{ "_setTraceClassAllocation", SetTraceClassAllocation,
+  Future<Success> setTraceClassAllocation(
+    String isolateId, {
+    String classId,
+    bool enable,
+  }) async {
+    final Map<String, dynamic> args = {};
+    if (classId != null) {
+      args['classId'] = classId;
+    }
+    if (enable != null) {
+      args['enable'] = enable ? true : false;
+    }
+
+    final response = await trackFuture(
+      '_setTraceClassAllocation',
+      callMethod('_setTraceClassAllocation', isolateId: isolateId, args: args),
+    );
+
+    return response as Success;
+  }
+
+  Future<CpuSamples> getAllocationSamples(
+    String isolateId, {
+    String classId,
+  }) async {
+    final Map<String, dynamic> args = {};
+    if (classId != null) {
+      args['classId'] = classId;
+    }
+    final response = await trackFuture(
+      '_getAllocationSamples',
+      callMethod('_getAllocationSamples', isolateId: isolateId, args: args),
```

Comment:
@bkonyi same issue with this API

Reply:
I also have a public implementation in the works for this.

Reply:
okay, ditto let me know when ready.